### DISCUSSION TITLE
MocaccinoOS upgrade > Fatal error

### DISCUSSION THREAD

**Discussion by Rocky-IV4** [mocaccinoOS/mocaccino#149] [Q&A] [answered]
Hello Team
Unfortunately Luet upgrade isn't successful after Macaccino iso is installed on old laptop.
[Fatal glibc error: CPU does not support x86-64-v2]
Can you propose a solution other than buying newer hardware?
Hardware is an old HP Intel Core2Duo machine.
Thanks
Rocky-IV

**Comment by joostruis** [comment]
Try to find some GNU/Linux distribution that is designed for the purpose of supporting very old hardware.

**Comment by joostruis** [comment]
Out of curiosity I wonder. How well did it perform on this old hardware?

**Comment by joostruis** [comment]
I've decided to undo v2.
https://github.com/mocaccinoOS/desktop/commit/222064e210c915b470b861930b47de2307242557

**Reply by joostruis** [reply]
FYI It will take a few days for this to become available in the repository, but yeah. You made me reconsider!

**Comment by joostruis** [comment, marked as answer]
Baseline is reset. You should be able to upgrade now.
